test/lint/lint-python.sh

```diff
@@ -102,7 +102,7 @@ if ! PYTHONWARNINGS="ignore" flake8 --ignore=B,C,E,F,I,N,W --select=$(IFS=","; e
     EXIT_CODE=1
 fi
 
-if ! mypy --ignore-missing-imports --show-error-codes $(git ls-files "test/functional/*.py" "contrib/devtools/*.py"); then
+if ! mypy --show-error-codes $(git ls-files "test/functional/*.py" "contrib/devtools/*.py"); then
```

Comment:
Something I've wondered about in the past is whether checking these in the same `mypy` invocation is a good idea. I think it will consider all the files part of the same top-level module, which might cause inconsistencies of different directories have same-named modules, for example.
That said, it works.